### PR TITLE
doc/rados: edit troubleshooting-mon.rst (1 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit the first 150 lines of
doc/rados/troubleshooting/troubleshooting-mon.rst.

https://tracker.ceph.com/issues/58485

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 1]
```diff
@@ -1,115 +1,128 @@
...
-  shame, but try to not lose a couple of hours looking for a deeper problem.
-  When running Kraken or later releases also ensure that the manager
-  daemons (``ceph-mgr``) are running, usually alongside each ``ceph-mon``.
- 
+  First, make sure that the monitor (*mon*) daemon processes (``ceph-mon``) are
+  running. Sometimes Ceph admins either forget to start the mons or forget to
+  restart the mons after an upgrade. Checking for this simple oversight can
+  save hours of painstaking troubleshooting. It is also important to make sure
+  that the manager daemons (``ceph-mgr``) are running. Remember that typical
+  cluster configurations provide one ``ceph-mgr`` for each ``ceph-mon``.
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:28]: Rook steadfastly will not run more than two mgrs.

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 2]
```diff
@@ -1,115 +1,128 @@
...
+
+**Can you reach the monitor nodes?**
+
+  In certain rare cases, there are ``iptables`` rules that block access to
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:32]: There may be ``iptables``

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 3]
```diff
@@ -1,115 +1,128 @@
...
+  monitor nodes or TCP ports. These rules might be left over from earlier
+  stress testing or rule development. To check for the presence of such rules,
+  SSH into the server and then try to connect to the monitor's ports
+  (``tcp/3300`` and ``tcp/6789``) using ``telnet``, ``nc``, or a similar tool.
 
-**Are you able to reach to the mon nodes?**
+**Does the ``ceph -s`` command run and receive a reply from the cluster?**
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:38]: Here and below we should probably be formal and suggest ``ceph status`` (same command)

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 4]
```diff
@@ -1,115 +1,128 @@
...
-Unix socket file. This file can be found in your monitor's ``run`` directory.
-By default, the admin socket will be kept in ``/var/run/ceph/ceph-mon.ID.asok``
-but this may be elsewhere if you have overridden the default directory. If you
-don't find it there, check your ``ceph.conf`` for an alternative path or
-run::
+A monitor's admin socket allows you to interact directly with a specific daemon
+by using a Unix socket file. This file is found in the monitor's ``run``
+directory. The admin socket's default directory is
+``/var/run/ceph/ceph-mon.ID.asok``, but this can be overridden and the admin
+socket might be elsewhere. If you cannot find it, either check your
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:80]: elsewhere, especially if your cluster's daemons are deployed in containers.

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 5]
```diff
@@ -1,115 +1,128 @@
...
-  ceph-conf --name mon.ID --show-config-value admin_socket
+   ceph-conf --name mon.ID --show-config-value admin_socket
 
-Bear in mind that the admin socket will be available only while the monitor
-daemon is running. When the monitor is properly shut down, the admin socket
-will be removed. If however the monitor is not running and the admin socket
-persists, it is likely that the monitor was improperly shut down.
-Regardless, if the monitor is not running, you will not be able to use the
-admin socket, with ``ceph`` likely returning ``Error 111: Connection Refused``.
+The admin socket is available for use only while the monitor daemon is running.
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:87]: s/while/when/
nitpick, "while" to me semi-infers that it exits normally from time to time

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:28]: Added.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst]: Accepted. Added.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 3 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst]: I've changed all instances of "ceph -s" to "ceph status".

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 4 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst]: Accepted. Changed.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 5 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst]: Accepted. Altered.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/51904 - Reef backport
https://github.com/ceph/ceph/pull/51905 - Quincy backport

**Pull request merged**